### PR TITLE
Tp add endpoint for current schema

### PR DESCRIPTION
**Pull request by timpeat**
Adds api endpoints for retrieving application and case_details schema in json-schema format.

Bumps version to 0.2.0

 curl http://laa-apply-for-criminal-legal-aid.test/api/schemata/application.json | jq

 curl http://laa-apply-for-criminal-legal-aid.test/api/schemata/case_details.json | jq

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -13,6 +13,7 @@ def matches?(_request)
   namespace :api, format: :json do
     scope module: :v1, constraints: ApiConstraint.new do
       resources :applications, format: :json, only: %i[index show]
+      resources :schemata, format: :json, only: %i[show]
```
Comment (willmcb) [inline config/routes.rb]: Do we have more than one schema? I'm not sure the plural adds anything given its a show action 🤔

Comment (willmcb) [reply config/routes.rb]: If we want to use the plural I think "schemas" is clearer.

**Review by willmcb — APPROVED** [review]
LGTM - tiny non-blocking comment on the schema plural :)

File: config/routes.rb (hunk 1 above)
Comment (timpeat) [reply config/routes.rb]: Yes I wondered the same. Will use schemas (I think we will have more).

**Pull request merged**